### ISSUE TITLE
First spline leg incorrect

### ISSUE THREAD

**Issue by sgofferj** [diydrones/apm_planner#727]
The first spline leg is at least displayed incorrectly. I didn't try flying it yet.

**Comment by sgofferj** [comment]
![screenshot-064](https://cloud.githubusercontent.com/assets/3258466/7979772/cde6be3c-0aaa-11e5-92d5-b0cccce59e04.png)

**Comment by billbonney** [comment]
I think the issue is that your HOME point and WP1 have to be in the same location. I'll take a look at a fix, but that's the workaround. HOME needs to be moveable again, as AC now supports a moveable home point

**Comment by AndKe** [comment]
Yes. Home should be movable. Also, it's a pain to have home at coordinates 0,0  - if 0,0  - then it should be ignored, or even made up near first item.
Whenever home spawns at 0,0  - for clean planning - it needs to be manually moved to realistic coordinates.
